templates/zerver/help/change-a-user's-name.md

```diff
@@ -0,0 +1,25 @@
+# Change a user's name
+Name of the user can be changed by the admin as well as user himself.
+
+Following are the steps to change the name of the user using an admin account:
+* Click on the ![cog](/static/images/help/cog.png) icon in the upper right corner
+of the right sidebar
+
+* Choose `Administration` in the ![admin-panel](/static/images/help/admin.png) panel.
```

Comment:
This renders kind of funny.  Can you say "in the panel below" and then put the image beneath this paragraph?